app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,16 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str, hummer_level: int) -> None:
+        super().__init__(nickname=nickname, favourite_dish=favourite_dish)
+        self._hummer_level = hummer_level
+
+    def get_rating(self) -> int:
+        return self._hummer_level + 4
+
+    def player_info(self) -> None:
+        return f"Dwarf warrior {self.nickname}. " \
+               f"{self.nickname} has a hummer of the " \
+               f"{self._hummer_level} level"

```

Comment:
Place multi-line string inside of `()` to avoid using line breaking operator `\`. Apply this change to all places